README.md

```diff
@@ -72,143 +75,29 @@ Directives from this repository are automatically tested with the following brow
+* Ask a question in [StackOverflow](http://stackoverflow.com/) under the [angular-ui-bootstrap](http://stackoverflow.com/questions/tagged/angular-ui-bootstrap) tag.
 
-```javascript
-grunt build:modal:tabs:alert:popover:dropdownToggle:buttons:progressbar
-```
-
-Check the Grunt build file for other tasks that are defined for this project.
-
-#### TDD
-* Run test: `grunt watch`
- 
-This will start Karma server and will continuously watch files in the project, executing tests upon every change.
+**Please do not create new issues in this repository to ask questions about using UI Bootstrap**
 
-#### Test coverage
-Add the `--coverage` option (e.g. `grunt test --coverage`, `grunt watch --coverage`) to see reports on the test coverage. These coverage reports are found in the coverage folder.
+## Found a bug?
+Please take a look at [CONTRIBUTING.md](CONTRIBUTING.md#you-think-youve-found-a-bug) and submit your issue [here](https://github.com/angular-ui/bootstrap/issues/new).
 
-### Customize templates
 
-As mentioned directives from this repository have all the markup externalized in templates. You might want to customize default
-templates to match your desired look & feel, add new functionality etc.
+----
 
-The easiest way to override an individual template is to use the `<script>` directive:
 
-```html
-<script id="template/alert/alert.html" type="text/ng-template">
-    <div class='alert' ng-class='type && "alert-" + type'>
-        <button ng-show='closeable' type='button' class='close' ng-click='close()'>Close</button>
-        <div ng-transclude></div>
-    </div>
-</script>
-```
-
-If you want to override more templates it makes sense to store them as individual files and feed the `$templateCache` from those partials.
-For people using Grunt as the build tool it can be easily done using the `grunt-html2js` plugin. You can also configure your own template url.
-Let's have a look:
-
-Your own template url is `views/partials/ui-bootstrap-tpls/alert/alert.html`.
-
-Add "html2js" task to your Gruntfile
-```javascript
-html2js: {
-  options: {
-    base: '.',
-    module: 'ui-templates',
-    rename: function (modulePath) {
-      var moduleName = modulePath.replace('app/views/partials/ui-bootstrap-tpls/', '');
-      return 'template/' + moduleName;
-    }
-  },
-  main: {
-    src: ['app/views/partials/ui-bootstrap-tpls/**/*.html'],
-    dest: '.tmp/ui-templates.js'
-  }
-}
-```
+# Contributing to the project
 
-Make sure to load your template.js file
-`<script src="/ui-templates.js"></script>`
-
-Inject the `ui-templates` module in your `app.js`
-```javascript
-angular.module('myApp', [
-  'ui.bootstrap',
-  'ui-templates'
-]);
-```
-
-Then it will work fine!
-
-For more information visit: https://github.com/karlgoldstein/grunt-html2js
+We are always looking for the quality contributions! Please check the [CONTRIBUTING.md](CONTRIBUTING.md) for the contribution guidelines.
 
-### Release
```

Comment:
Same here. Its in the wiki
https://github.com/angular-ui/bootstrap/wiki/Development